data_structures/binary_tree/avl_tree.py

```diff
@@ -11,15 +11,15 @@
 
 
 class my_queue:
```

Comment:
```suggestion
class MyQueue:
```
Proper Python class naming.  See CONTRIBUTING.md.

Reply:
I just implemented the static type checking on an exsisting document in the repo . But I would do the suggested changes .